### ISSUE TITLE
textarea not rendering error styling

### ISSUE THREAD

**Issue by TakeFlightGraphics** [running-coder/jquery-form-validation#10]
What needs to happen to get the textarea label and border to render error styling?

It is not working on the form I just built. Please advise.

View code here: http://www.svcommunityservices.org/volunteer-signup-form.html

Thanks!

**Comment by running-coder** [comment]
Hey TakeFlightGraphics,

thanks for using the plugin and providing a testing link it made the whole debug process a lot easier.

The issue is that on the textarea you have the NOTEMPTY validation but between the opening and closing tag there is a couple whitespaces (probably caused by a line return inside your code)

I've patched the NOTEMPTY rule to contain at least 1 non-space character (will be released in the next version)

```javascript
// Before
NOTEMPTY: /./
// After
NOTEMPTY: /\S/
```

meanwhile you can quickly patch your issue by putting the closing </textarea> tag next to the opening one to avoid white spaces

![Textarea NOTEMPTY](http://i.imgur.com/eXjhQhg.png)

**Comment by TakeFlightGraphics** [comment]
Thank you for checking out my issue. The above tweak fixed the label. It now turns red for errors. However, the border is still not rendering red for errors. Any ideas?

I did notice that this code:

div.error,
div.error-list,
label.error,
input.error,
select.error {
    color: #e13369 !important;
    border-color: #e13369 !important;
}

does not include a textarea selector. Could that be it?

Thanks!

**Comment by running-coder** [comment]
Yea! good catch

I've also noticed you are using MIXED for first name and last name validation. I checked the regex on my side and it might not be appropriated since it's missing apostrophe character ' and allows entries for numeric characters. (ex: James O'Reily will not be valid)

I've added a Rule (will go inside 1.5.2)
```javascript
NAME: /^['a-z\s-]+$/i
```

**Comment by TakeFlightGraphics** [comment]
Perfect. I updated the selector list:

div.error,
div.error-list,
label.error,
input.error,
select.error,
textarea.error {
    color: #e13369 !important;
    border-color: #e13369 !important;
}

and it now works. I will check out the NAME rule tomorrow for first and last name fields. Working on taxes right now. Fun, wow!

Thanks!

**Comment by TakeFlightGraphics** [comment]
Tom:

Where can I make a donation to you? I could swear I saw a donate button 
somewhere.

Please let me know. Thanks!

Best regards,

Robin Fisher
Take Flight Graphics~

**Comment by running-coder** [comment]
Hey,

I have currently no donation button but I might add one in the near future because it indeed helps for supporting the fixes and new features.

Thanks for your support and feedback!

ps. I've modified the above NAME rule to match also punctuated roman letters to cover more possibilities (released in 1.5.2)

```javascript
NAME: /^['a-zãàáäâẽèéëêìíïîõòóöôùúüûñç\s-]+$/i,
```
